ts/node/spell_check.ts

```diff
@@ -1,7 +1,8 @@
-import { BrowserWindow, Menu } from 'electron';
+import { type BrowserWindow, Menu } from 'electron';
 import { sync as osLocaleSync } from 'os-locale';
+import type { SetupI18nReturnType } from '../types/Localizer';
 
-export const setup = (browserWindow: BrowserWindow, messages: any) => {
+export const setup = (browserWindow: BrowserWindow, i18n: SetupI18nReturnType) => {
   const { session } = browserWindow.webContents;
   const userLocale = process.env.LANGUAGE
     ? process.env.LANGUAGE
```

Comment:
update: we actually don't 

If we were to do this, and because "en_AU" is not a supported locale on crowdin (and many others), any Australian word like "bogan", "arvo" or "esky" would be considered a mistake. Even if the user has set his locale to be en_AU and he has the right spell check tools installed